### ISSUE TITLE
`mantra create` should halt when already in a meteor project

### ISSUE THREAD

**Issue by sungwoncho** [mantrajs/mantra-cli#63]
Running `mantra create` should not create another meteor app while already inside a meteor app. Helpful when someone accidentally runs `mantra create` when she really means `mantra generate`.

**Comment by thancock20** [comment]
So, adding the following code to the top of the create function would work if you try to run `mantra create` in the root directory of the project:

```
  if (fs.existsSync(`${shelljs.pwd()}/.meteor`)) {
    console.log('You are already in a Meteor project');
    return;
  }
```

Would this be good enough? At least until there's a solution to #3?

**Comment by sungwoncho** [comment]
That seems like a good start. PR welcomed.

**Comment by sungwoncho** [comment]
Fixed in #103.
